php.md

```diff
@@ -23,3 +23,33 @@ PHP
     *   [PHPCDC](https://github.com/sebastianbergmann/phpdcd)
     *   [SensioLabs Security Checker](https://github.com/sensiolabs/security-checker)
     *   [PHP Mess Detector](http://phpmd.org)
+
+In-house conventions
```

Comment:
I think we should use title case for headers. I would also call it something like "Coding Style Guide" to be more specific, and mention that it's in addition to PSR-2.